linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -145,30 +154,43 @@ static void threadpool_dispose(THREADPOOL* threadpool)
 
 **SRS_THREADPOOL_LINUX_07_015: [** `threadpool_destroy` shall destroy the SRW lock by calling `srw_lock_destroy`. **]**
 
+[Will be moved from threadpool_close.]
+**S_R_S_THREADPOOL_LINUX_07_089: [** `threadpool_close` shall signal all threads `threadpool` is closing by calling `InterlockedHL_SetAndWakeAll`.  **]**
```

Comment:
>gnal all threads `threadpool [](http://example.com/codeflow?start=64&length=28)

Nitpick: consider rephrasing just a bit 
"signall all the threads that `threadpool` is closing ..."

Reply:
Ok

Reply:
Removed